blockchain/blockchain.go

```diff
@@ -624,6 +625,7 @@ func (bc *BlockChain) setHeadBeyondRoot(head uint64, root common.Hash, repair bo
 	bc.futureBlocks.Purge()
 	bc.db.ClearBlockChainCache()
 	//TODO-Klaytn add governance DB deletion logic.
+	bc.db.DeleteGovernanceIdx(head)
```

Comment:
Is this proper location to remove governance idx here? How about moving this to `delFn`?

Reply:
As we talked, if `repair` flag is not set, the deletion should be operated. So, it would be better to move this into `delFn`